src/components/common/form/error-summary/ErrorSummary.spec.jsx

```diff
@@ -1,69 +1,29 @@
 import { screen } from '@testing-library/react';
-import { renderWithTimecardContext } from '../../../../test/helpers/TimecardContext';
-
 import ErrorSummary from './ErrorSummary';
+import { renderWithApplicationContext } from '../../../../test/helpers/TestApplicationContext';
 
 describe('ErrorSummary', () => {
   window.HTMLElement.prototype.scrollIntoView = jest.fn();
   it('should render an error summary component with all error messages', () => {
-    const testErrors = {
-      test: { inputName: 'test', message: 'Date cannot be blank' },
-      'test-day': { inputName: 'test-day', message: 'Enter a day' },
-      'test-month': { inputName: 'test-month', message: 'Enter a month' },
-      'test-year': { inputName: 'test-year', message: 'Enter a year' },
-    };
-    renderWithTimecardContext(
-      <ErrorSummary
-        errors={testErrors}
-        keys={['test', 'test-day', 'test-month', 'test-year']}
-      />
-    );
-
-    const overallErrorMessage = screen.getByText('Date cannot be blank');
-    const dayErrorMessage = screen.getByText('Enter a day');
-    const monthErrorMessage = screen.getByText('Enter a month');
-    const yearErrorMessage = screen.getByText('Enter a year');
+    const testErrors = [];
+    testErrors.push({
+      key: 'invalidEnd',
+      inputName: 'shift-finish-time',
+      message: 'Test message for end time',
+      errorPriority: 2,
+    });
+    testErrors.push({
+      key: 'invalidStart',
+      inputName: 'shift-start-time',
+      message: 'Test message for start time',
+      errorPriority: 1,
+    });
```

Comment:
Can we initialise testErrors with the errors inside to save a couple of lines of code?

Reply:
Updated the test errors throughout the app